dashboard/app/models/script.rb

```diff
@@ -919,7 +920,7 @@ def self.setup(custom_files, new_suffix: nil, new_properties: {})
       added_scripts = scripts_to_add.sort_by.with_index {|args, idx| [args[0][:id] || Float::INFINITY, idx]}.map do |options, raw_lesson_groups|
         add_script(options, raw_lesson_groups, new_suffix: new_suffix, editor_experiment: new_properties[:editor_experiment])
       rescue => e
-        raise e, "Error adding script named '#{options[:name]}': #{e}"
+        raise e, "Error adding script named '#{options[:name]}': #{e}", e.backtrace
```

Comment:
for my own information, did this end up being helpful? based on the PR description where I added this, it looks like I was getting the backtraces already: https://github.com/code-dot-org/code-dot-org/pull/35660 but I'm not clear based on my output whether some of it just became part of the "message" part of the exception.

Reply:
I posted my before and after above, not sure why I was getting a different result from you. I can dig into this a bit more later.